app/screens/auth/SwitchNetwork.tsx

```diff
@@ -77,7 +84,7 @@ const SwitchNetwork = ({ history, location }: AuthRouterParams) => {
       : [{ label: 'NO NETWORKS AVAILABLE', isDisabled: true }];
 
   const goNext = (genesisID: string | undefined) => {
-    const { creatingWallet, isWalletOnly, mnemonic } = location.state;
+    const { creatingWallet, isWalletOnly, mnemonic } = location?.state || {};
```

Comment:
Will be looking much better if we will place this get operation above  , out of function and will replace all places with location?.state?.PROPERTY
```
const { creatingWallet, isWalletOnly, mnemonic } = location?.state || {};
```